.gitignore

```diff
@@ -0,0 +1 @@
+venv/
```

Comment:
Always add a .gitignore so that you don't commit unnecessary files, like venv folders for example!